Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -8,7 +8,7 @@
 DEFAULT_LIMIT = 100
 
 # Standard headers
-HEADERS = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+HEADERS = {'Content-Type': 'application/json', 'Accept': '*/*'}
```

Comment:
I have changed because iac/v2/scans returns {
    "timestamp": "2021-10-12T08:28:22.804+00:00",
    "path": "/v2/scans",
    "status": 406,
    "error": "Not Acceptable",
    "message": "",
    "requestId": "d5712ff2-157729"
} if I set the Accept header to "application/json"